configuration/klippy/beacon_mesh.py

```
@@ -33,142 +151,1771 @@ def register_handler(self):
+				return False
+
+		return True
 
 	#####
 	# Beacon Scan Compensation
 	#####
-	def compensate_beacon_scan(self, profile):
-		systime = self.reactor.monotonic()
+	def apply_scan_compensation(self, comp_mesh_profile_name) -> bool:
+		if not comp_mesh_profile_name:
+			raise TypeError("Argument comp_mesh_profile_name must be provided")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)